packages/dropdowns/src/views/menu/StyledItemAnchor.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { componentStyles } from '@zendeskgarden/react-theming';
+import { StyledItemTypeIcon } from './StyledItemTypeIcon';
+import { StyledOption } from '../combobox/StyledOption';
+
+const COMPONENT_ID = 'dropdowns.menu.item_anchor';
+
+export const StyledItemAnchor = styled(StyledOption).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  as: 'a'
+})`
+  direction: ${props => props.theme.rtl && 'rtl'};
+  text-decoration: none;
+  color: unset;
+  &[aria-current='page'] > ${StyledItemTypeIcon} {
```

Comment:
```suggestion

  &[aria-current='page'] > ${StyledItemTypeIcon} {
```

[nit] always nice to add a space before styling blocks